crates/hc_bundle/tests/test_cli.rs

```diff
@@ -324,6 +312,7 @@ fn test_all_dna_manifests_match_schema() {
 }
 
 #[test]
+#[cfg(not(feature = "unstable-migration"))]
```

Comment:
Removing this test in the `unstable-migration` case since it would require having a dedicated json schema just for this feature for the test to pass.

Reply:
Might be good to duplicate this test with and without the feature.